code/datums/station_traits/positive_traits.dm

```diff
@@ -217,3 +217,39 @@
 	var/obj/item/implant/deathrattle/implant_to_give = new()
 	deathrattle_group.register(implant_to_give)
 	implant_to_give.implant(living_mob, living_mob, TRUE, TRUE)
+
+
+/datum/station_trait/wallets
+	name = "Wallets"
+	trait_type = STATION_TRAIT_POSITIVE
+	show_in_report = TRUE
+	weight = 10
+	report_message = "It has become temporarily fashionable to use a wallet, so everyone on the station has been issued one."
+	force = TRUE // XXX remove before merging/committing
```

Comment:
Note to other maintainers: Do not merge this without having this changed to FALSE.

Reply:
well then remove it :P